pkg/storage/image.go

```diff
@@ -253,8 +253,6 @@ func (svc *imageService) ListImages(systemContext *types.SystemContext, filter s
 			if err != nil {
 				return nil, err
 			}
-		} else {
-			//return nil, err
```

Comment:
This was commented out in 4cddd75130b.  I don't understand what was going on there, but "swallowing error" makes me suspect we want a placeholder issue if we remove this comment.

Reply:
https://github.com/kubernetes-incubator/cri-o/issues/1548